BlueprintUI/Sources/BlueprintView/BlueprintView.swift

```diff
@@ -277,6 +280,8 @@ public final class BlueprintView: UIView {
                 appearanceTransitionsEnabled: hasUpdatedViewHierarchy
             )
         )
+        
+        self.onDidLayoutContents?()
```

Comment:
Can you move this all the way to the bottom, after the flags are reset and after signpost logging? Or use `defer`?

Reply:
Ahh good call